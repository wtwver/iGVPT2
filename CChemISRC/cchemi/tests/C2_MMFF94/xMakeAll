#!/bin/bash
echo " Begin QFF2MR-VPT2 calculation">  $0.log

CCHEMI=/home/theochem/allouche/tmp/CChemI-050116/cchemi/cchemi
export PATH=$PATH:/home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94

# Opt+ Freq => C2Freq.gab
$CCHEMI C2.ici >> $0.log 2>  $0.err

# generate QFF energies files
/bin/rm -f C2VQFF_*.ici
/bin/rm -f C2VQFF_*.gab
$CCHEMI C2V.ici >> $0.log 2>  $0.err
ls C2VQFF_* > xp
sed -i 's/C2VQFF/rCC C2VQFF/' xp
#cat xp
chmod u+x xp
./xp >>  $0.log 2>> $0.err
sed -i 's/RunType=GenerateQFF2MRFiles/#RunType=GenerateQFF2MRFiles/g' C2V.ici
sed -i 's/#RunType=ComputeQFF2MRFromFiles/RunType=ComputeQFF2MRFromFiles/g' C2V.ici

# generate txt data for VPT2 calculation
$CCHEMI C2V.ici  >>  $0.log 2>> $0.err

sed -i 's/#RunType=GenerateQFF2MRFiles/RunType=GenerateQFF2MRFiles/g' C2V.ici
sed -i 's/RunType=ComputeQFF2MRFromFiles/#RunType=ComputeQFF2MRFromFiles/g' C2V.ici

# make ici  file for VPT2 calculation
echo 'RunType=VPT2' > C2VQFF.ici
cat C2VQFF.txt >> C2VQFF.ici

# generate txt data for VPT2 calculation
$CCHEMI C2VQFF.ici  >  $0.out 2>> $0.err


#/bin/rm xp
#/bin/rm C2VQFF_*.ici
#/bin/rm C2VQFF_*.gab
echo "==========================================================================="
echo "See $0.out , $0.log and $0.err files "
echo "==========================================================================="
