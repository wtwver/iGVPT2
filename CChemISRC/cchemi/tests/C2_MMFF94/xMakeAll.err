
real	0m0.093s
user	0m0.028s
sys	0m0.000s

real	0m0.075s
user	0m0.028s
sys	0m0.000s

real	0m0.085s
user	0m0.024s
sys	0m0.000s

real	0m0.089s
user	0m0.024s
sys	0m0.000s

real	0m0.046s
user	0m0.020s
sys	0m0.004s

real	0m0.098s
user	0m0.024s
sys	0m0.000s

real	0m0.075s
user	0m0.024s
sys	0m0.000s
