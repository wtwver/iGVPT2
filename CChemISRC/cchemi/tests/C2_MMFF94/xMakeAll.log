 Begin QFF2MR-VPT2 calculation
seed = -1
----------------------------------------------------------
runType=OPTFREQ
model=OPENBABEL
----------------------------------------------------------

type 1
Establishing connectivity : 2 connections...
Establishing connectivity : 3 connections...
Establishing connectivity : 4 connections...
Establishing connectivity : non bonded ...
We use the internal optimizer
type 1
Establishing connectivity : 2 connections...
Establishing connectivity : 3 connections...
Establishing connectivity : 4 connections...
Establishing connectivity : non bonded ...
#Number of free coordinates =1 
Minimization by QuasiNewton method
Gradient = 167.99639857660932 energy = 3.97268976972905 
Gradient = 30188.39563485311737 Energy(kcal/mol) = 6745.07354383551956 
Gradient = 162.83192125908383 Energy(kcal/mol) = 3.69829398597737 
Gradient = 3.72350670115371 Energy(kcal/mol) = 0.00151442544850 
Gradient = 0.75845072633442 Energy(kcal/mol) = 0.00006318417252 
Gradient = 0.00261890394211 Energy(kcal/mol) = 0.00000000075264 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000000000000 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000123443473 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000170173756 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000181882117 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000184453205 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185000917 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185116884 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185142640 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Line search failed. See documentation of routine mcsrch.
 Error return of line search: info = 3 Possible causes:
 function or gradient are incorrect, or incorrect tolerances.
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Gradient = 0.00000185149077 Energy(kcal/mol) = 0.00000000000000 
Optimized geometry saved in /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2Opt.gab file
Save molecule in /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2Opt.gab
Gradient Norm  = 0.00000185149077 energy = 0.00000000000000(kcal/mol)
#Number of free coordinates =1 
Computing of IR spectra... Please wait
nAtoms = 2
Gradient Norm  = 0.00000184936460 energy = 0.00000000000000(kcal/mol)
Frequencies and modes in the /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2Freq.gab file
Save molecule in /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2Freq.gab
Save molecule in /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2Freq.gab
type 1
Establishing connectivity : 2 connections...
Establishing connectivity : 3 connections...
Establishing connectivity : 4 connections...
Establishing connectivity : non bonded ...
#Number of free coordinates =1 
========================================================================================================================
# Geometry and velocities
# Fixed normal modes (ground, n=0) and scaled to T = 300.000000
# Geometry, nAtoms, charge, spin multiplicity.
# symbol, MMType, pdbType, residueName, numResidue, charge, layer, variable, x(Ang),y,z, nconn, num1, typ1, num2, typ2,...
Geometry
2 0 1
 C C C DUM 0 0.000000000000 2 1 0.000000000000 0.000000000000 0.644909332716 1  2 3
 C C C DUM 0 0.000000000000 2 1 0.000000000000 0.000000000000 -0.644909332716 1  1 3
# Velocities, vx(Ang/AKMA-time) vy(Ang/AKMA-time) vz(Ang/AKMA-time): 1fs = 0.020455 AKMA
Velocities
0.000000000000e+00 0.000000000000e+00 -1.576067654005e-01
0.000000000000e+00 0.000000000000e+00 1.576067654005e-01
========================================================================================================================

seed = -1
----------------------------------------------------------
runType=GENERATEQFF2MRFILES
model=OPENBABEL
----------------------------------------------------------

type 5
Establishing connectivity : 2 connections...
Establishing connectivity : 3 connections...
Establishing connectivity : 4 connections...
Establishing connectivity : non bonded ...
delta = 0.500000 reducedCoordiantes
nAtoms = 2
nModes = 1
see C2V.iciQFF_*.ici generated files

seed = -1
----------------------------------------------------------
runType=COMPUTEQFF2MRFROMFILES
model=OPENBABEL
----------------------------------------------------------

Reading molecule from /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2VQFF_0.gab file
nAtoms = 2
fileName = /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2VQFF_1.ici
buffer = #RunType=Gradient

buffer = RunType=Energy

buffer = #RunType=ComputeQFF2MRFromFiles

buffer = Model=OpenBabel

buffer = openBabelCommand=obgradient -ff mmff94

buffer = 

buffer = dx=0.01

buffer = 

buffer = mopacCommand=mopac

buffer = orcaCommand=runOrca

buffer = fireflyCommand=firefly

buffer = gaussianCommand=runGauss

buffer = genericCommand=runGeneric

buffer = 

buffer = 

buffer = 

buffer = #Mode: Freq= 2116.501790000000 Mass= 12.000000000000 Q= Qeq + 0.103276957357 akI=-1.369961

fileName = /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2VQFF_2.ici
buffer = #RunType=Gradient

buffer = RunType=Energy

buffer = #RunType=ComputeQFF2MRFromFiles

buffer = Model=OpenBabel

buffer = openBabelCommand=obgradient -ff mmff94

buffer = 

buffer = dx=0.01

buffer = 

buffer = mopacCommand=mopac

buffer = orcaCommand=runOrca

buffer = fireflyCommand=firefly

buffer = gaussianCommand=runGauss

buffer = genericCommand=runGeneric

buffer = 

buffer = 

buffer = 

buffer = #Mode: Freq= 2116.501790000000 Mass= 12.000000000000 Q= Qeq + 0.068851304905 akI=-1.369961

fileName = /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2VQFF_3.ici
buffer = #RunType=Gradient

buffer = RunType=Energy

buffer = #RunType=ComputeQFF2MRFromFiles

buffer = Model=OpenBabel

buffer = openBabelCommand=obgradient -ff mmff94

buffer = 

buffer = dx=0.01

buffer = 

buffer = mopacCommand=mopac

buffer = orcaCommand=runOrca

buffer = fireflyCommand=firefly

buffer = gaussianCommand=runGauss

buffer = genericCommand=runGeneric

buffer = 

buffer = 

buffer = 

buffer = #Mode: Freq= 2116.501790000000 Mass= 12.000000000000 Q= Qeq + 0.034425652452 akI=-1.369961

fileName = /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2VQFF_4.ici
buffer = #RunType=Gradient

buffer = RunType=Energy

buffer = #RunType=ComputeQFF2MRFromFiles

buffer = Model=OpenBabel

buffer = openBabelCommand=obgradient -ff mmff94

buffer = 

buffer = dx=0.01

buffer = 

buffer = mopacCommand=mopac

buffer = orcaCommand=runOrca

buffer = fireflyCommand=firefly

buffer = gaussianCommand=runGauss

buffer = genericCommand=runGeneric

buffer = 

buffer = 

buffer = 

buffer = #Mode: Freq= 2116.501790000000 Mass= 12.000000000000 Q= Qeq - 0.034425652452 akI=-1.369961

fileName = /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2VQFF_5.ici
buffer = #RunType=Gradient

buffer = RunType=Energy

buffer = #RunType=ComputeQFF2MRFromFiles

buffer = Model=OpenBabel

buffer = openBabelCommand=obgradient -ff mmff94

buffer = 

buffer = dx=0.01

buffer = 

buffer = mopacCommand=mopac

buffer = orcaCommand=runOrca

buffer = fireflyCommand=firefly

buffer = gaussianCommand=runGauss

buffer = genericCommand=runGeneric

buffer = 

buffer = 

buffer = 

buffer = #Mode: Freq= 2116.501790000000 Mass= 12.000000000000 Q= Qeq - 0.068851304905 akI=-1.369961

fileName = /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2VQFF_6.ici
buffer = #RunType=Gradient

buffer = RunType=Energy

buffer = #RunType=ComputeQFF2MRFromFiles

buffer = Model=OpenBabel

buffer = openBabelCommand=obgradient -ff mmff94

buffer = 

buffer = dx=0.01

buffer = 

buffer = mopacCommand=mopac

buffer = orcaCommand=runOrca

buffer = fireflyCommand=firefly

buffer = gaussianCommand=runGauss

buffer = genericCommand=runGeneric

buffer = 

buffer = 

buffer = 

buffer = #Mode: Freq= 2116.501790000000 Mass= 12.000000000000 Q= Qeq - 0.103276957357 akI=-1.369961

fileName = /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2VQFF_7.ici
numberOfFrequencies = 1
QFF parameters saved in /home/theochem/allouche/tmp/CChemI-050116/cchemi/tests/C2_MMFF94/C2VQFF.txt file

