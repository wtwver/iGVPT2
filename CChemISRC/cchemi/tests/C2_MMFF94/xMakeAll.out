seed = -1
----------------------------------------------------------
runType=VPT2
model=GVPT2
----------------------------------------------------------

nFrequencies=1
VPT2 model : GVPT2

Gradients
END

Hessian
1 1      2116.5017900000
END

Cubic
1 1 1       654.0278610000
END

Quartic
1 1 1 1       157.2647740000
END

Coriolis
END

Rotation constants
1         1.0000000000
2         1.0000000000
3         1.0000000000
END
nFrequencies=1
nDim=3
VPT2 Properties model : GVPT2 : Gaussian method

First derivatives
END


Second derivatives
END


Cubic derivatives
END

nFrequencies 1
nStates 3
State n 1	0 
State n 2	1 
State n 3	2 

Xkl (cm^-1) by VPT2+K model
1 1       -11.2234006957
END

C=0.001143

Diagonal of H (cm^-1) by VPT2+K model
     1055.4450448261         0.0000000000 
     3149.5000334346      2094.0549886085  1(1) 
     5221.1082206516      4165.6631758256  1(2) 
END


Resonance Matrix (cm^-1) by VPT2+K model
========================================
 Fermi Resonances terms :
 0 Fermi Resonances 
----------------------------------------------------------------------------------------------------

 Darling-Dennison Resonances terms :
 0 Darling-Dennison Resonances 
----------------------------------------------------------------------------------------------------

END



Results
========================
       E(harm)      E(anharm)     Mode(Quanta), Coef 
       -------      ---------     ---------------- 
    0.00000000     0.00000000     [ 1.000] 
 2116.50179000  2094.05498861     [ 1.000  1(1)] 
 4233.00358000  4165.66317583     [ 1.000  1(2)] 



Results With intensities
========================
       E(harm)      E(anharm)        I(harm)      I(anharm)     Coef Mode(Quanta) 
       -------      ---------        -------      ---------     ---------------- 
    0.00000000     0.00000000     0.00000000     0.00000000     [ 1.000] 
 2116.50179000  2094.05498861     0.00000000     0.00000000     [ 1.000  1(1)] 
 4233.00358000  4165.66317583     0.00000000     0.00000000     [ 1.000  1(2)] 



Results With intensities
========================

Fundamental Bands
-----------------
       E(harm)      E(anharm)        I(harm)      I(anharm)       Coef. Mode(Quanta) 
  ------------   ------------   ------------   ------------     -------------------- 
 2116.50179000  2094.05498861     0.00000000     0.00000000     [ 1.000  1(1)] 

Overtones
---------
       E(harm)      E(anharm)        I(harm)      I(anharm)       Coef. Mode(Quanta) 
  ------------   ------------   ------------   ------------     -------------------- 
 4233.00358000  4165.66317583     0.00000000     0.00000000     [ 1.000  1(2)] 

Combination Bands
----------------
       E(harm)      E(anharm)        I(harm)      I(anharm)       Coef. Mode(Quanta) 
  ------------   ------------   ------------   ------------     -------------------- 


